prosper_nn/models/hcnn_compressed/hcnn_compressed.py

```diff
@@ -0,0 +1,357 @@
+import torch.nn as nn
+import torch
+from typing import Optional, Type
+from . import hcnn_cell_compressed
+
+
+class HCNN_compressed(nn.Module):
+    """
+    The HCNN_compressed class creates a Historical Consistent Neural Network that makes a difference
+    between the features you are really interested in (task) and the features you use to improve
+    the forecast of the task variables (support).
+    It is possible to compress the support variables before they are compared with the expectation
+    in the model. In that way, the number of features in the target layers get decreased.
+
+    A Historical Consistent Neural Network belongs to the class of Recurrent Neural Networks.
+    A special feature of the architecture is that it has no input in the common sense.
+    Instead, all the inputs are also interpreted as targets that are equally important.
+    So the architecture does not distinguish between input and output features.
+    Furthermore, it uses teacher forcing to improve the hidden state.
```

Comment:
Is this necessary here? I find it a bit confusing, because it basically says the opposite thing in the paragraph before.